hack/rapture/README.md

```diff
@@ -104,7 +104,11 @@ The entire build process takes several hours. Once you are ready to begin, the d
 
 ### Validating packages
 
-Now that `rapture` has successfully complete, we need to verify the packages that were just created. This validation can be done on any instance where Kubernetes is not already installed. (Ideally, you would want to spin up a new VM to test.)
+Now that `rapture` has successfully complete, we need to verify the packages that were just published and created correctly.
+
+To check for publish success, see the [Debian package list][deb-package-list] and [RPM package list][rpm-package-list] for the versions that were just uploaded.
```

Comment:
probably also include a less manual way?

```
curl https://packages.cloud.google.com/apt/dists/kubernetes-xenial/main/binary-amd64/Packages | grep $version
```

and


```
curl https://packages.cloud.google.com/yum/repos/kubernetes-el7-x86_64/repodata/primary.xml | grep $version
```

Reply:
Thanks will do